examples/networking/subnet/main-customdns.tf

```diff
@@ -0,0 +1,8 @@
+resource "vkcs_networking_subnet" "subnet-with-dns" {
+  name       = "subnet-with-dns-tf-example"
+  network_id = vkcs_networking_network.app.id
+# here we set DNS'es instead of built in ones. 
```

Comment:
Format this like comments in other examples

Reply:
changed